velox/functions/prestosql/InPredicate.cpp

```diff
@@ -19,21 +19,124 @@
 namespace facebook::velox::functions {
 namespace {
 
+// Returns NULL if
```

Comment:
do you know if this Presto-specific, or that's actual ansi sql semantic?

Reply:
I don't know. The SQL spec is not publicly available.